### ISSUE TITLE
Tune Hyperparameters

### ISSUE THREAD

**Issue by 7138gmhampton** [7138gmhampton/EmotionDetection#11]
- [x] Consider automagic approach a la Keras Tuner
- [x] Batch Size - Note effective range
- [x] Scale down
- [x] No of filter features
- [x] Dropout - if needed
